docs/03-assertions.md

```diff
@@ -137,6 +137,10 @@ Assert that `actual` is deeply equal to `expected`. See [Concordance](https://gi
 
 Assert that `actual` is not deeply equal to `expected`. The inverse of `.deepEqual()`. Returns a boolean indicating whether the assertion passed.
 
+### `.unorderedEqual(actual, expected, message?)`
+
+Assert that all values in `actual` are in `expected`, returning a boolean indicating whether the assertion passed.
```

Comment:
Could you clarify the differences with `deepEqual`?

Reply:
Thoughts?

> Assert that all values in `actual` are in `expected`. This is a variant of `.deepEqual()` that does not depend
on the order of `actual`/`expected` and only compares instances of `Map`s or `Set`s/arrays.
>
> The size of `actual` and `expected` must be equal. For `Map`s, each key-value pair in `actual` must be in
`expected`, and vice-versa. For `Set`s/arrays, each value in `actual` must be in `expected`.
>
> Returns `true` if the assertion passed and throws otherwise.